api/main_endpoints/routes/OfficeAccessCard.js

```diff
@@ -44,6 +45,20 @@ function checkIfCardExists(cardBytes) {
   });
 }
 
+let clients = [];
+
+const response = {
+  endpoint: '/verify',
+  statusCode: 200,
+  message: 'Card authorized!'
+};
+
+const writeRequestResponse = () => {
```

Comment:
response shouldnt be global, instead something you initialize before you call `writeRequestResponse` and pass as a parameter
```suggestion

const writeRequestResponse = (response) => {
```

Reply:
done